### PR TITLE
renames after david's suggestions

### PR DESCRIPTION
**Pull request by gimer**

### CODE REVIEW

**Comment by rg911** [comment]
@gimer should we update the all.cat for the new schemas?

File: schemas/coresystem/voting_key_link.cats [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# binary layout for a non-embedded voting key link transaction
+struct VotingKeyLinkTransaction
+	const uint8 version = 1
+	const EntityType entityType = 0x4243
+
+	inline Transaction
+	inline VotingKeyLinkTransactionBody
+
+# binary layout for an embedded voting key link transaction
+struct EmbeddedVotingLinkKeyTransaction
```
Comment (Jaguar0625) [inline schemas/coresystem/voting_key_link.cats:20]: KeyLink

File: schemas/coresystem/voting_key_link.cats [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import "transaction.cats"
+
+# binary layout for a voting key link transaction
+struct VotingKeyLinkTransactionBody
+	# linked account public key
```
Comment (Jaguar0625) [inline schemas/coresystem/voting_key_link.cats:5]: ~~account~~

**Comment by gimer** [comment]
> If so, there is also a missing import (`import "resolution_statement/resolution_statements.cats"`) which the PR was closed unmerged.

which PR was that?

(clearly this shouldn't be done within *this* PR, if possible PRs should be self-contained)

**Comment by rg911** [comment]
> > If so, there is also a missing import (`import "resolution_statement/resolution_statements.cats"`) which the PR was closed unmerged.
> 
> which PR was that?
> 
> (clearly this shouldn't be done within _this_ PR, if possible PRs should be self-contained)

This one https://github.com/nemtech/catbuffer/pull/22

p.s. but not sure we should include the account states in or not.

**Pull request merged**